graphs/cmumps-patient.fbp

```diff
@@ -0,0 +1,31 @@
+# cmumps-patient.fbp
+
+# Takes a patient ID and returns a string.
```

Comment:
Suggest adding a little more comment: What is the purpose of the returned string?  What does it represent?